continuation-token/src/main/kotlin/net/sprd/common/continuationtoken/Pagination.kt

```diff
@@ -77,16 +77,16 @@ private fun <P : Pageable> checksumsAreUnequal(previousToken: ContinuationToken,
     return previousToken.checksum != createTokenFromEntities(checksumSlice)?.checksum
 }
 
-private fun <P : Pageable> createEmptyPage(): Page<P> = Page(listOf(), null)
-private fun <P : Pageable> createLastPage(entities: List<P>): Page<P> = Page(entities, null)
+private fun <P : Pageable> createEmptyPage(token: ContinuationToken?): Page<P> = Page(listOf(), token, false)
```

Comment:
Can we use a namend arguments for the boolean argument to increase the readability?

Reply:
For sure, will fix that.